src/models/order.py

```diff
@@ -112,6 +112,7 @@ def from_dict(cls, order_id: str, data: OrderSerializedType) -> Order:
             "buy_amount",
             "is_sell_order",
             "allow_partial_fill",
+            "is_liquidity_order",
```

Comment:
Are you sure this is a required field?

Reply:
No I am not sure. Just did not feel any less required than other fields.

Reply:
Being required here means that it is always expected to be provided in an orders data from the instance provided. It's probably best to check this with the instances and their specification.

Reply:
After a bit of thinking: I believe it is required. As a solver, if that was not present in an order then I am not sure what to do with the order. A solver treats `"is_liquidity_order": true` different from `"is_liquidity_order": false`